practice/completer/site.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * Completer object contains completions.
+ *
+ * completions will have keys as the completion and values as weights.
+ */
+function Completer() {
+  this.completions = {};
+}
+
+/**
+ * modifyCompleter has these methods:
+ *
+ * addCompletion() adds a completion to the completions container (in-place)
+ * removeCompletion() removes a completion from the container (in-place)
+ * complete() returns an array of completions, highest weighted first
+ * selectCompletion() increases the weight of a given completion (in-place)
+ *
+ */
+var modifyCompleter = {
+  addCompletion: function(str) {
+    this.completions[str] = 1;
+  },
+  removeCompletion: function(str) {
+    delete this.completions[str];
+  },
+  complete: function(prefix) {
+    var re = RegExp('^' + prefix);
```

Comment:
I'm not sure what the variable name "re" is referring to. Is there a clearer variable name to explain what is being stored in this variable, like "testPrefix"?